imports/api/Constants.js

```diff
@@ -34,7 +37,21 @@ export const Constants = {
 		// Japanese Riichi Style
 		JAPANESE: "jpn"
 	},
-	MAHJONG_CLUB_LEAGUE: "Mahjong Club League"
+
+	// Cardinal winds
+	DIRECTIONS: {
```

Comment:
I said this in another pull request review.  If you're going to refactor constants/formatting/style please do so in a seperate branch.